dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/Node.java

```diff
@@ -157,4 +158,32 @@ public void setAssignmentContext(AssignmentContext assignmentContext) {
         }
         return transitiveOperands;
     }
+
+    /**
+     * Returns a verbose printed representation of this, useful for debugging.
```

Comment:
For consistency, in a few places in this PR, I would replace the `printed` with `string`.

Reply:
I agree.  That was for consistency with existing methods' documentation, but I have changed those too.